page.mdx

```diff
@@ -91,9 +101,36 @@ iconType: "solid"
 ---
 ```
 
+## Wide Mode
+
+You can hide the table of contents on the right side of the page. This is useful
```

Comment:
```suggestion
You can hide the table of contents on the left side of the page. This is useful
```

Reply:
table of contents is this thing
![CleanShot 2024-03-28 at 10 42 20@2x](https://github.com/mintlify/docs/assets/55263191/99121f3e-ab1a-437d-ab71-0a768249e717)

Reply:
Ohh right right my b